### PR TITLE
Create 2023-06-26-Embracing-Kanban.md

### PR DESCRIPTION
**Pull request by andreboscatto**
Initial Draft

### CODE REVIEW

File: _posts/2023-06-26-Embracing-Kanban.md [hunk 1]
```diff
@@ -6,11 +6,16 @@ categories: agile
 ---
 
 # Introduction
+In our quest to improve our workflow and ensuring a great product providing a better customer experience, we have made a significant decision to move away from traditional sprint planning and the old-fashioned waterfall. Instead, we have embraced Kanban, a methodology that gives us the flexibility we need to handle the demands of support scenarios. This change allows us to address urgent tasks promptly without being tied to strict sprint schedules, ultimately putting our customers at the forefront.
```
Comment (ikerexxe) [inline _posts/2023-06-26-Embracing-Kanban.md]: We weren't following the `old-fashioned waterfall` model, so I'd remove that part.

File: _posts/2023-06-26-Embracing-Kanban.md [hunk 2]
```diff
@@ -6,11 +6,16 @@ categories: agile
...
 
 # Conclusion
+In conclusion, our decision to embrace Kanban and move away from fixed timelines and the traditional waterfall delivery model marks a significant shift in our approach. By adopting Kanban, we are able to invite early feedback, create an adaptable environment, and ultimately strive for increased productivity. Kanban introduces a more flexible approach to our support-oriented scenarios, where we prioritize adaptability and focus on meeting customer needs. Through this transition, we aim to enhance our support processes, deliver faster outcomes, and cultivate a customer-centric approach that sets us on the path to success. Nevertheless, we have strong confidence that this shift allows us to move away from frustration and work towards meeting expectations effectively.
```
Comment (ikerexxe) [inline _posts/2023-06-26-Embracing-Kanban.md]: You are mentioning again the waterfall model here.

File: _posts/2023-06-26-Embracing-Kanban.md (hunk 1 above)
Comment (andreboscatto) [reply _posts/2023-06-26-Embracing-Kanban.md]: In the end of the day, we have a huge waterfall mindset behind the scenes, it is why I mentioned that, maybe I should have done it differently

**Review by ikerexxe — APPROVED** [review]
LGTM!

**Pull request closed without merging**